validator/node/node.go

```diff
@@ -134,17 +134,29 @@ func NewValidatorClient(cliCtx *cli.Context) (*ValidatorClient, error) {
 	return validatorClient, nil
 }
 
-func newRouter(cliCtx *cli.Context) *mux.Router {
+// m is a middleware type
+type m func(http.Handler) http.Handler
```

Comment:
this type can be added to the api package, but it would probably be good to check if some type like this already exists